### DISCUSSION TITLE
Unable to move the mapping tracker

### DISCUSSION THREAD

**Discussion by Traenol** [Lailloken/Exile-UI#283] [Q&A] [answered]
So, this is a strange one, and I can't seem to figure it out. I am unable to move the mapping tracker(not the toggle button, but the one that shows the current map). Currently running 1.50.2, tried running as admin, restarting, doing a completely fresh run of LLK(just the basics to let me start it) to no avail. Shut down every running app aside from PoE and AHK to no avail. Script permissions test all comes back yes(though I doubt that is relevant). I am out of ideas at this point, and wondering if there is something I am doing wrong,

**Comment by Lailloken** [comment]
The wiki will always state if ui-elements in a feature are movable. The tracker panel isn't. It's supposed to be in the corner between exp bar and the open inventory (because that's what you're usually looking at in between maps), so double-checking the loot-tracking or the kill-tracking becomes easier.

It's possible that it gets misplaced at some specific resolutions, so let me know if that's the issue. Out of curiosity, where would you put it if you were able to move it?

**Comment by Traenol** [comment]
So this is where it currently is, just a tad annoying that it is covering the xp bar, and was wanting to move it up to where I would expect it to be. 
![tracker](https://github.com/Lailloken/Lailloken-UI/assets/142574525/4827510f-ab78-4082-bc1b-f181f832c146)
Its probably misplaced if its not supposed to be movable, the wording in the wiki made it sound like both were movable, but that could just be my interpretation. I'm running in windowed fullscreen, 2560x1440, LLKUI has the same setting.

Edit: just a quick edit, its perfectly aligned with the inventory panel open on its right side, just realized I didn't have it open when I grabbed the screen

**Reply by Lailloken** [reply, marked as answer]
Okay, maybe "in the corner between exp-bar and inventory" on the wiki needs to be changed. I simply chose the lesser of two evils: Right now, it covers a fraction of the exp bar. But if I move it up slightly, it covers the hideout-edit button and makes it unclickable. One can still hover the exp-bar to see the exact %-value, but you wouldn't be able to get around the button being blocked.

It's also harder to hit the exact edge of the exp-bar, so the panel would float one or two pixels above the exp-bar or clip it by one or two pixels (because of scaling and rounding pixel-values), which drives me nuts. Moreover, the panel is hidden while mapping, and that's where one mostly watches the exp-bar and its movement.

**Comment by Traenol** [comment]
Well, that answers the only minor issue I have had with the tool(which is excellent btw). Certainly not a deal breaker, as its rarely actually obscuring anything useful. I wouldn't mind being able to move it if you ever want to enable it to be though.
